deny.toml

```diff
@@ -116,6 +116,9 @@ skip = [
     { name = "rand_core", version = "0.6.2" },
     { name = "reqwest", version = "0.11.24" },
     { name = "rustls-pemfile", version = "1.0.4" },
+    # Used by reqwest
+    { name = "system-configuration", version = "0.5.1" },
+    { name = "system-configuration-sys", version = "0.5.0" },
```

Comment:
what changed here? is there a way to tweak the dependencies to remove this duplication instead?